### ISSUE TITLE
Error if notifier terminates earlier than source

### ISSUE THREAD

**Issue by anilanar** [franciscotln/callbag-take-until#7]
The following marble test fails because the implementation assumes notifier talkback will be available when source terminates.

```
it('works if notifier terminates earlier than source', () =>
  pipe(
    listenable('a-b-c|'),
    takeUntil(listenable('|')),
    willBe('a-b-c|'),
  ));
```

**Comment by franciscotln** [comment]
Hi @anilanar, thanks for submitting this issue. I'll take a look into that. Sorry for the late response.

**Comment by franciscotln** [comment]
@anilanar that's fixed by [this commit](https://github.com/franciscotln/callbag-take-until/commit/67b17b3b77f0f0720fed2e6475037be274b793eb) and v3.0.0 was just published.
